tests/testthat/test-oauth-cache.R

```diff
@@ -39,3 +53,30 @@ test_that("token saved to and restored from cache", {
   expect_equal(token_b$endpoint, token_a$endpoint)
   expect_equal(token_b$credentials, token_a$credentials)
 })
+
+test_that("new caches are Rbuildignored and gitignored", {
+  owd <- setwd(tmp_dir())
+  on.exit(setwd(owd))
+  old <- options()
```

Comment:
I just copied the sole pre-existing instance of this pattern verbatim. I can do this  ☝️ instead.